frontend/app-development/features/dataModelling/SchemaEditorWithToolbar/SchemaGenerationErrorsPanel.tsx

```diff
@@ -0,0 +1,57 @@
+import classes from './SchemaGenerationErrorsPanel.module.css';
+import React from 'react';
+import { Alert, Button, ErrorMessage, Paragraph } from '@digdir/design-system-react';
+import { Trans, useTranslation } from 'react-i18next';
+import { XMarkIcon } from '@navikt/aksel-icons';
+
+export interface SchemaGenerationErrorsPanelProps {
+  onCloseErrorsPanel: () => void;
+  schemaGenerationErrorMessages: string[];
+}
+
+export const SchemaGenerationErrorsPanel = ({
+  onCloseErrorsPanel,
+  schemaGenerationErrorMessages,
+}: SchemaGenerationErrorsPanelProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Alert severity='danger'>
+      <div className={classes.errorPanel}>
+        <div>
+          <Paragraph>{t('api_errors.DM_01')}</Paragraph>
+          <ul>
+            {schemaGenerationErrorMessages?.map((errorMessage, index) => {
+              return (
+                <li key={index}>
```

Comment:
We should add the `errorMessage` to the index as well since it's not recommended to use the index only for keys within React. More info about this can be read here: https://react.dev/learn/rendering-lists#why-does-react-need-keys 


```suggestion
                <li key={`${errorMessage}-index`}>
```